concat.js

```diff
@@ -109,15 +109,17 @@ module.exports = class Concat extends Plugin {
 
   _doPatchBasedBuild(patch) {
     if (!this.concat) {
-      this.concat = new this.Strategy(merge(this.sourceMapConfig, {
+      let options = merge(this.sourceMapConfig, {
         separator: this.separator,
         header: this.header,
         headerFiles: this.headerFiles,
         footerFiles: this.footerFiles,
         footer: this.footer,
         contentLimit: this.contentLimit,
         baseDir: this.inputPaths[0]
-      }));
+      });
+      options['fs'] = this.input;
```

Comment:
why a computed property`['fs']` over a regular property `options.fs = this.input;`

Reply:
I am not sure why I did it. I have changed it to `option.fs`